Update source config migration guide (#1682)
wso2/docs-choreo-dev: en/docs/develop-components/manage-component-source-configurations.md

| **context**          | Required     | The context (base path) of the API that Choreo exposes via this endpoint.        |
| **schemaFilePath**   | Required     | The swagger definition file path. Defaults to the wildcard route if not provided. This field should be a relative path to the project path when using the **Java**, **Python**, **NodeJS**, **Go**, **PHP**, **Ruby**, and **WSO2 MI** buildpacks. For REST endpoint types, when using the **Ballerina** or **Dockerfile** buildpack, this field should be a relative path to the component root or Docker context.|

## Update to the latest source configuration file

> With the upcoming deprecation of the `component-config.yaml` file and the `endpoints.yaml` file, Choreo provides a seamless process to update to the latest `component.yaml` file. Follow these steps to transition seamlessly:

Reviewer: ```suggestion
With the upcoming deprecation of the `component-config.yaml` file and the `endpoints.yaml` file, Choreo provides a seamless process to move to the latest version of the `component.yaml` file. Follow these steps to transition seamlessly:
```

### Step 1: Identify whether your current source configuration file is outdated

You can determine whether you’re using an outdated configuration file in the following ways:

- **During a build:**
     - On the **Build** page, after triggering a build, a warning will appear if your source configuration file is outdated. Click **View Details** in the **Initialization** -> **Source Config Validation** step. You will see an information banner with an option to update the file.

- **In existing deployments:**
     - If an existing deployment uses a build with an outdated source configuration file, you will see a warning icon on the build card of the relevant environment card.

### Step 2: Generate the latest `component.yaml` file

- In either of the scenarios above, click **Update** to initiate the process of generating the latest `component.yaml` file. The updated file will retain your existing configurations to ensure a smooth update process.

### Step 3: Add the generated configuration file to your repository

1. Download the generated `component.yaml` file.

> 2. Replace the existing file in the `.choreo` folder within your source repository.

Reviewer: ```suggestion
2. Replace the existing source configuration file in the `.choreo` folder within your source repository.
```

    - Ensure the file name is `component.yaml`.

### Step 4: Trigger a new build

> 1. Commit the new `component.yaml` file to your repository.

Reviewer: ```suggestion
1. Commit the new `component.yaml` file to your repository. Push changes to the remote Git repository.
```

2. Trigger a build using the latest commit.

### Step 5: Verify the update

Once the build is complete:

  - Confirm that the warning messages no longer appear.

>   - You can benefit from the latest features and enhancements provided by the updated source configuration file.

Reviewer: Shall we ask the user to deploy the changes to complete the process?

Author: Updated like this
```
Once the build is complete:

  - Deploy the build.
  - Confirm that the warning messages no longer appear.

You can benefit from the latest features and enhancements provided by the updated source configuration file.
```
